Add openapi-mode configuration option (#412)
PostgREST/postgrest-docs: configuration.rst

    server-unix-socket-mode = "660"

.. _openapi-mode:

openapi-mode
------------

  Specifies how the OpenAPI output should be displayed:

  .. code:: bash

>     # Follows the access control for the JWT role.
>     # Shows information depending on the permissions that the user making the request has

Reviewer: "role" seems more clear than "user":
```suggestion
    # Follows the privileges of the JWT role claim
    # Shows information depending on the permissions that the role making the request has
```

    openapi-mode = "follow-privileges"

>     # Ignores the access control for the JWT role.
>     # Shows all the exposed information, regardless of the permissions the user making the request has

Reviewer: Ditto. Also, "privileges" is more precise than "access control"([doc](https://www.postgresql.org/docs/13/ddl-priv.html#PRIVILEGE-ABBREVS-TABLE))
```suggestion
    # Ignores the privileges of the JWT role claim
    # Shows all the exposed information, regardless of the permissions the role making the request has
```
